src/test/kotlin/calculator/domain/StringCalculatorTest.kt

```diff
@@ -0,0 +1,61 @@
+package calculator.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatExceptionOfType
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullAndEmptySource
+import org.junit.jupiter.params.provider.ValueSource
+
+class StringAddCalculatorTest {
+    private lateinit var calculator: StringAddCalculator
+
+    @BeforeEach
+    fun setUp() {
+        calculator = StringAddCalculator()
+    }
+
+    @DisplayName(value = "빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.")
+    @ParameterizedTest
+    @NullAndEmptySource
+    fun emptyOrNull(text: String?) {
+        assertThat(calculator.add(text)).isZero
+    }
+
+    @DisplayName(value = "숫자 하나를 문자열로 입력할 경우 해당 숫자를 반환한다.")
+    @ParameterizedTest
+    @ValueSource(strings = ["1"])
+    fun oneNumber(text: String) {
+        assertThat(calculator.add(text)).isSameAs(Integer.parseInt(text))
```

Comment:
```suggestion
        assertThat(calculator.add(text)).isSameAs(text.toInt())
```

Integer.parseInt는 자바 클래스를 사용해요. 

코틀린 답게 사용하는 방법 중에 하나는 바로 자바 클래스를 사용하지 않고, 코틀린에 있는 수많은 클래스와 메서드를 사용하는 게 한 가지 방법이라고 생각해요.